packages/ui/src/components/Table/column-sort.composable.ts

```diff
@@ -0,0 +1,124 @@
+import { Ref } from 'vue'
+import { ColumnApi } from './api'
+import { ColumnSortOrder, TableRow } from './types'
+import { get } from 'lodash-es'
+
+export function useColumnSort(data: Ref<TableRow[]>) {
```

Comment:
Would be nice to add **unit** tests for sorting, because it seems quite complex.